apps/console/src/features/private-key-jwt/pages/private-key-jwt-config-edit.tsx

```diff
@@ -0,0 +1,250 @@
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+    AlertLevels, IdentifiableComponentInterface
+} from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import {
+    DocumentationLink,
+    GridLayout,
+    PageLayout,
+    useDocumentation
+} from "@wso2is/react-components";
+import { AxiosError } from "axios";
+import React, {
+    FunctionComponent,
+    ReactElement,
+    SyntheticEvent,
+    useEffect,
+    useState
+} from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Dispatch } from "redux";
+import { Checkbox, CheckboxProps, Grid, Icon, Message } from "semantic-ui-react";
+import { AppConstants, history } from "../../core";
+import { ServerConfigurationsConstants } from "../../server-configurations";
+import { updateJWTConfig, useTokenReuseConfigData } from "../api";
+
+/**
+ * Private key JWT client authentication for OIDC configuration page.
+ *
+ * @param props - Props injected to the component.
+ * @returns Private key JWT client authentication for OIDC configuration page component.
+ */
+export const PrivateKeyJWTConfigEditPage: FunctionComponent<IdentifiableComponentInterface> = (
+    props: IdentifiableComponentInterface
+): ReactElement => {
+    const { [ "data-componentid" ]: componentId } = props;
+
+    const dispatch: Dispatch = useDispatch();
+    const { t } = useTranslation();
+    const { getLink } = useDocumentation();
+    const [ enableTokenReuse, setEnableTokenReuse ] = useState<boolean>(true);
+
+    const {
+        data: tokenReuseData,
+        isLoading: isLoading
+    } = useTokenReuseConfigData();
+
+    useEffect(() => {
+        if (tokenReuseData) {
+            setEnableTokenReuse(tokenReuseData?.enableTokenReuse);
+        }
+    }, []);
+
+    /**
+     * Handles back button click event
+     */
+    const handleBackButtonClick = () => {
+        history.push(
+            AppConstants.getPaths()
+                .get("GOVERNANCE_CONNECTOR")
+                .replace(
+                    ":id",
+                    ServerConfigurationsConstants.LOGIN_ATTEMPT_SECURITY_CONNECTOR_CATEGORY_ID
+                )
+        );
+    };
+
+    const resolveConnectorUpdateErrorMessage = (error: any): string => {
+        
+        return (
+            t("console:manage.features.jwtPrivateKeyConfiguration.notifications.error.description",
+                { description: error.response.data.description })
+        );
+    };
+
+    const resolveConnectorUpdateSuccessMessage = (): string => {
+
+        return (
+            t("console:manage.features.jwtPrivateKeyConfiguration.notifications.success.description")
+        );
+    };
```

Comment:
Do we need this function as it is only used in one place and we can directly use its return value inside the success alert call in line 104?